package.json

```diff
@@ -61,13 +61,12 @@
     "url": "https://github.com/harness/ff-nodejs-server-sdk"
   },
   "dependencies": {
-    "@types/lru-cache": "^7.10.10",
     "axios": "^0.22.0",
     "axios-retry": "^3.2.0",
     "jwt-decode": "^3.1.2",
     "keyv": "^4.0.3",
     "keyv-file": "^0.2.0",
-    "lru-cache": "^6.0.0",
+    "lru-cache": "^7.0.0",
```

Comment:
we could probably go to the highest minor version for 7 here (8 required node v16). I'll update and test on Monday